### PR TITLE
Upgrading.rst (Fix Duplicate statement)

### PR DESCRIPTION
**Pull request by Eshy10**
In this PR worked on:

- Where ``<version>`` is the version of JupyterHub you are upgrading to statement occurs twice. 
Screenshot 2022-10-14 at 03 11 26

Fix: Remove Duplicate
Screenshot 2022-10-14 at 03 10 42

- Add a colon to the explanation of the environments:

Screenshot 2022-10-14 at 03 13 34

Fix:
Screenshot 2022-10-14 at 03 10 42

### CODE REVIEW

File: docs/source/admin/upgrading.rst [hunk 1]
```diff
@@ -63,11 +63,11 @@ Upgrade JupyterHub packages
 
 There are two environments where the ``jupyterhub`` package is installed:
 
-#. The *hub environment*, which is where the JupyterHub server process
+#. The *hub environment*: Which is where the JupyterHub server process
```
Comment (minrk) [inline docs/source/admin/upgrading.rst:66]: A colon would be _instead of_ `which is...`:

```suggestion
#. The *hub environment*: where the JupyterHub server process
```

We also don't typically capitalize after a colon unless it's a full sentence (which this isn't).

File: docs/source/admin/upgrading.rst [hunk 2]
```diff
@@ -63,11 +63,11 @@ Upgrade JupyterHub packages
...
    runs. This is started with the ``jupyterhub`` command, and is what
    people generally think of as JupyterHub.
 
-#. The *notebook user environments*. This is where the user notebook
+#. The *notebook user environments*: This is where the user notebook
```
Comment (minrk) [inline docs/source/admin/upgrading.rst:70]: Same here. If using a colon, drop `This is`

File: docs/source/admin/upgrading.rst [hunk 3]
```diff
@@ -92,8 +92,6 @@ with:
 
    conda install -c conda-forge jupyterhub==<version>
 
-Where ``<version>`` is the version of JupyterHub you are upgrading to.
```
Comment (minrk) [inline docs/source/admin/upgrading.rst:95]: 👍

**Review by minrk — COMMENTED** [review]
Thanks for your contribution! I've made some suggestions inline.

File: docs/source/admin/upgrading.rst (hunk 2 above)
Comment (Eshy10) [reply docs/source/admin/upgrading.rst]: I have made changes @minrk

File: docs/source/admin/upgrading.rst (hunk 1 above)
Comment (Eshy10) [reply docs/source/admin/upgrading.rst]: I have made changes on this @minrk

**Comment by minrk** [comment]
Great, thank you!

**Pull request merged**